app/perfsymboltable.cpp

```diff
@@ -282,10 +282,10 @@ int PerfSymbolTable::parseDie(CuDieRangeMapping *cudie, Dwarf_Die *top, quint64
         location.parentLocationId = parentLocationId;
 
         int callLocationId = m_unwind->resolveLocation(location);
-        return insertSubprogram(cudie, top, entry, offset, size, relAddr, binaryId, binaryPathId, actualPathId, callLocationId, isKernel);
+        return insertSubprogram(cudie, top, entry, offset, size, relAddr, binaryId, binaryPathId, actualPathId, callLocationId, isKernel, true);
```

Comment:
are you sure this is the right place to make the inline vs. non-inline decision? see `PerfSymbolTable::parseDwarf` which gets a clear list of inlined DIEs and the subprogram die - I think that's the level where this should be handled, no?